cczoo/cross_lang_framework/install cr_lan framework .bash

```diff
@@ -0,0 +1,164 @@
+function version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
```

Comment:
Please remove the space in the file name.
install cr_lan framework .bash -> install cr_lan framework.bash